docs/team/leeenen.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Lee En En's Project Portfolio Page
+---
+
+### Project: TAlent Assistant™
```

Comment:
Copy pasta this too 🍝 

Sorry I should've added this first before the description, missed it.